.github/dependabot.yml

```diff
@@ -64,6 +64,15 @@ updates:
     schedule:
       interval: weekly
       day: sunday
+  - package-ecosystem: gomod
+    directory: /design/log
```

Comment:
Should we start this in the directory we think the API or SDK will end up in (e.g. /log) so we don't have to migrate later on?  We can make it clear that it is in the design stage.

Reply:
I intentionally do not want to put the code in the expected destinations.

Once the design is approved, I think it will be a lot easier to review PRs that would target a different destination instead of modifying the prototype.

The design should define what would be the expected Go modules. 

We will remove the `/design` directory after the actual implementation is completed and prototype is no longer needed.

Reply:
It will be better to place everything in a separate repository.